### DISCUSSION TITLE
PatchTool 6.1 4/16

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#140] [Announcements]
- 修正 #131 #138 #139 
- 縮減程式（JRE）大小

檔案：
- FFXIVChnTextPatch-0416.rar：本體。一般而言，下載這個即可。

選用：
- [第三方4K字體包](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.53/font-4k.rar)：部分解析度設定下會出現字體錯位等情形，請自行照需求使用。解壓後改名為font放到resource資料夾下（取代原資料夾）。
- rawexd-zhcn-0416.rar：簡體中文CSV。解壓後放到resource資料夾下（取代原資料夾）。
- [FFXIV-jar-0211.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.56/FFXIV-jar-0211.rar)：如果你有遇到JNI error，請參考 #90 放到與exe同位置使用。
- [backup-0415.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.59/backup-0415.rar)：乾淨的0x0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原（或是手動取代遊戲檔案），然後再次嘗試漢化。
- [rawexd-ja-610-v2-0415.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.59/rawexd-ja-610-v2-0415.rar)：日文原文的CSV。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v2.60'>PatchTool 6.1 4/16</a>.</em>

**Comment by 0c123** [comment]
招募面板裡，伺服器內的招募無法顯示當前地點
![image](https://user-images.githubusercontent.com/103809902/163663466-0e4ddea5-f46b-4a53-99d3-fadbbfc8a009.png)

**Reply by GpointChen** [reply]
了解，我找時間看看！

**Comment by qwe3410475** [comment]
感謝大大辛苦更新~

**Comment by siler2011** [comment]
感謝大大!!

**Comment by Billy-MC** [comment]
感謝大大, 6.01 萬魔殿還沒有漢化的對嗎

**Comment by cydad111** [comment]
请问如何下载更快呢- -正常谷歌下载没4小时下载不完(还是目前这种容量,之前那400MB的要半天- -),但是过1小时多后都会自动失败,要求授权

**Comment by KaiBaake** [comment]
![image](https://user-images.githubusercontent.com/104083752/164253704-17dc0ad3-7e6b-44d1-b565-898d18245ce4.png)
請問這是正常的嗎?

**Reply by peckcat** [reply]
是正常的

**Reply by KaiBaake** [reply]
這任務名稱就是這樣嗎? 還以為發生甚麼事

**Reply by peckcat** [reply]
對，他是一系列

**Comment by SadsaucR** [comment]
謝謝大大 辛苦了 FF14有你更好

**Comment by chiami911** [comment]
昨天在解15等主線任務"サスタシャに挑む者"的時候，與任務中NPC對話發現卡住，且無法繼續進行對話的現象，推測可能是6.1新版本有針對舊文本更動對話數量。

感謝大大持續更新~
